website/blog/2019-01-21-7.3.0.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title:  "7.3.0 Released: Named capturing groups, private instance accessors and smart pipelines"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2019-01-21 16:00:00
+categories: announcements
+share_text: "Babel 7.3.0 Released"
+---
+
+7.3.0 Released: Named capturing groups, private instance accessors and smart pipelines
+===
+
+After over 80 commits, the latest Babel minor release is here!
+
+This release includes support for named capturing groups in regular expressions, private instance accessors, the smart pipeline operator and a bunch of improvements to TypeScript parsing. You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.3.0).
+
+<!-- truncate -->
+
+> TODO: new contributors
```

Comment:
☝️

Reply:
Yup, I'll add them when generating the changelog